lua/fff/rust/file_picker.rs

```diff
@@ -317,11 +318,12 @@ impl FilePicker {
 
                 file.update_frecency_scores();
             }
-        };
+        }
 
-        Ok(self.get_cached_files())
+        self.get_cached_files()
```

Comment:
just for reference sometimes I leave the functions return result if I know that I might need result later, it is anyway fused by a compiler if there a single pass used